src/transformStateWithClones.js

```diff
@@ -65,7 +65,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  let copy = { ...state };
+
+  for (const transform of transforms) {
+    switch (transform.operation) {
```

Comment:
usually in switch case construction developers adds default case, it's a common issue that there is no default case, but it would be nice to have something like 
```
default: 
  // Do nothing
```

or 
```
default: 
 Error.throw('there is no such a case')
```